// protobuf.UpdateClientInfo.UpdateClientInfoReqIdl
syntax = "proto3";

message UpdateClientInfoReqIdl {
    string cuid = 1;
    message DataReq {
        string bduss = 1;
        string device = 2;
        bytes secretKey = 3;
        int32 width = 6;
        int32 height = 7;
        string stoken = 12;
    }
    DataReq data = 2;
}
